Library/Homebrew/config.rb

```diff
@@ -39,4 +39,4 @@
 HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
 
 # Load path used by standalone scripts to access the Homebrew code base
-HOMEBREW_LOAD_PATH = HOMEBREW_LIBRARY_PATH
+HOMEBREW_LOAD_PATH = HOMEBREW_LIBRARY_PATH.to_s
```

Comment:
I'm not sure I understand. Unless I'm missing a piece of the puzzle, I'm simply converting a `Pathname` instance to a string (this should yield a new `String` instance) and assigning it to a new constant, that didn't previously exist. How is that going to cause a warning?